.travis.yml

```diff
@@ -34,10 +34,10 @@ before_install:
 
 install:
   - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then export PATH="/usr/local/opt/gettext/bin:$PATH"; fi
-  # Minimum requirements: CMake 3.8.2, Boost 1.64
+  # Minimum requirements: CMake 3.9, Boost 1.64
```

Comment:
we should write that into the README.md too

Reply:
We don't have any versions in the readme, so I would not add it there. As usual: "Documentation is always out of date", so I prefer code be the docu. It is right at the top of the CML and running CMake will give a very clear error if it is to low. It is only a comment here to explain the below line(s) (why that versions were chosen and not any other)